### ISSUE TITLE
BUG - sev-3 - All - App v2.22 introduced an analytics event of "(other)"

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#8082]
## What happened?

Seems like we introduced a bug in analytics events in 2.22 (which went live today). We’re logging an event of `(other)` a lot. It only is happening today, and only with v2.22, so......

<img width="1364" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOEdVQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--cd4356d6cefa840290bb4c77e93ad20e0ff7347e/image.png" alt="image.png" />

It's definitely not the intended analytic event (or maybe not an intended event at all) so we should either replace it with what it's supposed to be, or remove it from our code.

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the [bug epic](https://app.zenhub.com/workspaces/va-mobile-product-view-610035bc5395bb000e62e529/issues/gh/department-of-veterans-affairs/va-mobile-app/5367)

**Comment by dumathane** [comment]
Took a look at this briefly today. First I checked to see if there were any analytic anomalies...but the way we reference the Events.<thing> seems consistent and our linter catches typos. The next thing I saw was there is one analytic event that appears to not include the () after (vama_gender_id_help) but that is managing to log fine. After that the only thing that seemed odd was how we are doing vama_claims_status_tab/vama_claim_details_tab but both of those have been logging since launch as well.
